package.json

```diff
@@ -82,7 +82,7 @@
       "node-forge@<1.3.0": ">=1.3.0",
       "glob-parent@<5.1.2": ">=5.1.2",
       "nth-check@<2.0.1": ">=2.0.1",
-      "@nucypher/nucypher-core": "0.13.0-alpha.1"
+      "@nucypher/nucypher-core": "^0.14.1"
```

Comment:
Judging by https://github.com/nucypher/nucypher-core/pull/89 there are no concerns about `0.14.1` not being compatible with nodes running `0.13.0` correct? The breaking changes were limited to the wasm bindings, which the nodes don't care about.

Reply:
Yes, there changes are strictly for `nucypher-core-wasm` and there is no impact on Python users.